### PR TITLE
Add Strix Music

### PR DESCRIPTION
**Pull request by y0lopix**
###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/microsoft/winget-pkgs/pull/1409)

### CODE REVIEW

File: manifests/ArloG/StrixMusic/1.4.7.0.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Id: ArloG.StrixMusic
+Version: 1.4.7.0
+Name: Strix Music
+Publisher: Arlo G.
+License: custom
```
Comment (samiranshah) [inline manifests/ArloG/StrixMusic/1.4.7.0.yaml:5]: @KevinLaMS Do we need more license info here?

**Review by ghost — APPROVED** [review]
Validation has completed

**Comment by Arlodotexe** [comment]
@y0lopix Hi there, Strix Music dev here. I see that this PR is closed, but due to the URL you gave it, it isn't getting the update to 1.5.0. 

It may be better to point it to the .appinstaller instead (https://arlodotappx.blob.core.windows.net/strixmusicstable/StrixMusic.appinstaller)

**Comment by y0lopix** [comment]
> @y0lopix Hi there, Strix Music dev here. I see that this PR is closed, but due to the URL you gave it, it isn't getting the update to 1.5.0.
> 
> It may be better to point it to the .appinstaller instead (https://arlodotappx.blob.core.windows.net/strixmusicstable/StrixMusic.appinstaller)

Hi, AppInstaller files are still unsupported by WinGet and each version should correspond to a single manifest. Also, dependencies management is not supported so all Strix Music manifests have been removed (#19964 #19965 #19966).

**Pull request merged**